documentation/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Troubleshooting Kubernetes vSphere Cloud Provider
+---
+This document is aimed to help debug issues in the Kuberentes vSphere Cloud Provider.
```

Comment:
Why not remove this line? Isn't it redundant?

Reply:
removed.